config/routes.rb

```diff
@@ -266,7 +266,11 @@
       post "/choose-provider-npq", to: "payment_breakdowns#choose_provider_npq", as: :choose_provider_npq
     end
     namespace :ecf do
-      resources :payment_breakdowns, only: %i[show]
+      resources :payment_breakdowns, only: %i[show payable] do
```

Comment:
@asmega can we change this to:

```ruby
resources :ecf_lead_prodviders, path: 'payment-breakdowns, only: %I[show payable], controller: :payment_breakowns do
   resource :payable, only: :show
end
```

Reply:
i agree the url structure is not the best, however i think it's ok for now. i think we should rejig all this when we come do all the other views/months/reports (or whatever they're called)